src/core/format/pretty-print.ts

```diff
@@ -1,4 +1,4 @@
-import markdown from "prettier/parser-markdown";
+import markdown from "prettier/plugins/markdown";
 import prettier from "prettier/standalone";
```

Comment:
Thanks for keeping things fresh @klappradla. 💚 

I only had a quick look, but it seems like you might need to update this import:

```suggestion
import * as prettier from "prettier/standalone";
```

🍋

Reply:
Yup 💜 

Already been there, but not quite dialed in yet. The interface also change to async...